src/backup

```diff
@@ -7,6 +7,7 @@ cleanup() {
   find /backups/ -type f -not -name ".*" -mtime +$CLEANUP_OLDER_THAN -exec rm {} \;
 }
 
+if [ -z "$MYSQL_ENV_MYSQL_HOST" ]; then echo "Error: MYSQL_ENV_MYSQL_HOST not set"; echo "Finished: FAILURE"; exit 1; fi
```

Comment:
It should not break if the variable is not set, instead it should use "mysql" as default